js/components/contactList/contactList_controller.js

```diff
@@ -92,10 +139,8 @@ angular.module('contactsApp')
 					}
 				});
 			}
-			// No contact previously loaded, let's load the first of the list if not in mobile mode
-			if(ctrl.loading && $(window).width() > 768) {
-				ctrl.setSelectedId(ctrl.contactList[0].uid());
-			}
+			var firstNames = ctrl.contactList.slice(0, 20).map(function (c) { return c.displayName(); });
+			ContactService.getFullContacts(firstNames);
```

Comment:
Strange, it changes the url with an uid and then a contact.
Anyway, we need to keep the old mechanic in place.
1. If mobile, do not load contact details and saty on the contacts list EXCEPT if there is a matching uid specified in the url
2. If not in mobile, load the first contact